src/images/features/bdc-color-logo.svg

```diff
@@ -0,0 +1 @@
5h10.06l5.92,12.8,5.97-12.8h9.91l-11.38,22.05Zm40.4,2.73c0,6.88-5.31,11.13-12.9,11.13-5.26,0-10.27-1.62-14.41-5.21l4.85-6.62c2.88,2.38,6.17,3.99,9.61,3.99,2.43,0,3.69-1.06,3.69-2.58v-.05c0-1.42-1.06-2.28-5.21-3.74-6.67-2.28-11.38-4.7-11.38-11.13v-.15c0-6.42,4.9-10.92,12.64-10.92,4.76,0,8.95,1.31,12.64,4.3l-4.6,6.78c-2.43-1.87-5.36-3.24-8.24-3.24-2.07,0-3.24,1.01-3.24,2.33v.05c0,1.57,1.11,2.38,5.61,4.05,6.73,2.17,10.92,4.9,10.92,10.92v.1Zm31.15-16.59h-8.75v27.21h-9.1v-27.21h-8.75v-8.19h26.6v8.19Z"/><polygon class="cls-1" points="307.25 68.59 313.82 68.59 310.53 58.23 307.25 68.59"/><path class="cls-1" d="m177.92,35.93v57.24h290.22l23.35-28.62-23.35-28.62H177.92Zm36.84,46.93c-9.05,0-16.08-6.93-16.08-17.96v-.61c0-10.92,7.13-18.05,16.44-18.05,5.56,0,9.36,2.18,12.24,5.56l-5.87,6.22c-1.52-1.82-3.44-3.34-6.22-3.34-4.15,0-7.23,3.69-7.23,9.71v.2c0,6.32,3.14,9.81,7.23,9.81,3.03,0,4.86-1.62,6.47-3.49l5.76,5.86c-2.88,3.54-6.83,6.07-12.74,6.07m41.32-.61l-2.07-6.37h-11.13l-2.02,6.37h-9.36l12.44-35.6h9.2l12.49,35.6h-9.56Zm36.81-27.21h-8.75v27.21h-9.1v-27.21h-8.75v-8.19h26.6v8.19Zm25.29,27.21l-2.07-6.37h-11.13l-2.02,6.37h-9.36l12.44-35.6h9.2l12.49,35.6h-9.56Zm38.58,0h-23.01v-35.4h9.1v27.21h13.91v8.19Zm19.06-13.35v13.35h-9.1v-13.25l-11.38-22.15h10.06l5.92,12.8,5.97-12.8h9.91l-11.38,22.05Zm40.4,2.73c0,6.88-5.31,11.13-12.9,11.13-5.26,0-10.27-1.62-14.41-5.21l4.85-6.62c2.88,2.38,6.17,3.99,9.61,3.99,2.43,0,3.69-1.06,3.69-2.58v-.05c0-1.42-1.06-2.28-5.21-3.74-6.67-2.28-11.38-4.7-11.38-11.13v-.15c0-6.42,4.9-10.92,12.64-10.92,4.76,0,8.95,1.31,12.64,4.3l-4.6,6.78c-2.43-1.87-5.36-3.24-8.24-3.24-2.07,0-3.24,1.01-3.24,2.33v.05c0,1.57,1.11,2.38,5.61,4.05,6.73,2.17,10.92,4.9,10.92,10.92v.1Zm31.15-16.59h-8.75v27.21h-9.1v-27.21h-8.75v-8.19h26.6v8.19Z"/></g></g><g id="xrushB.tif"><g><path class="cls-2" d="m487.54,93.83h-.47s-.03,0-.04,0c-.11,0-.21-.02-.32-.03-.41-.04-.8-.14-1.19-.28-.81-.3-1.5-.77-2.07-1.41-.78-.88-1.22-1.91-1.31-3.09-.04-.53,0-1.05.13-1.57.29-1.17.9-2.13,1.85-2.87.85-.67,1.82-1.03,2.9-1.09.56-.03,1.1.02,1.64.17,1.35.38,2.38,1.17,3.1,2.37.4.68.63,1.41.69,2.2,0,.06.01.12.02.18v.47s0,.04,0,.06c0,.11-.02.23-.03.34-.05.43-.15.85-.32,1.25-.49,1.23-1.33,2.14-2.51,2.74-.6.3-1.23.48-1.9.53-.06,0-.12.01-.18.02Zm-4.36-5.17c0,2.27,1.84,4.12,4.12,4.12,2.27,0,4.12-1.84,4.12-4.12,0-2.27-1.84-4.12-4.12-4.12-2.27,0-4.12,1.84-4.12,4.12Z"/><path class="cls-2" d="m488.2,88.72c.22.06.4.18.55.35.09.11.18.23.25.35.15.28.29.57.44.85.16.31.31.62.46.93,0,.02.01.03.02.05h-.05c-.46,0-.93,0-1.39,0-.04,0-.05-.01-.07-.04-.22-.46-.45-.91-.68-1.37-.08-.16-.17-.31-.31-.43-.12-.11-.28-.18-.44-.19-.18-.01-.36,0-.54-.01,0,0,0,0-.02,0v2.04h-1.39v-5.41s.03,0,.04,0c.73,0,1.46,0,2.2,0,.36,0,.73.04,1.07.16.25.09.47.22.63.44.12.17.19.36.22.56.04.28.04.56-.04.83-.11.38-.37.63-.72.79-.08.03-.16.06-.24.09Zm-1.77-.48c.24,0,.47,0,.7,0,.08,0,.15-.01.23-.03.22-.05.38-.16.44-.38.05-.19.05-.37,0-.56-.05-.17-.15-.29-.32-.34-.09-.03-.19-.05-.28-.06-.25-.01-.49,0-.74-.01,0,0-.01,0-.02,0v1.38Z"/></g></g></g></g></svg>
```

Comment:
this might have gotten wrapped in somehow, but we have the fixed bdc logo in the #7 branch